about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Science Festival Site</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <!-- HEADER -->
+    <header class="header">
```

Comment:
- [ ] Make sure the desktop version is divided into a top dark bar and a bottom white bar for both `home `and `about `pages.
Also, the top short bar section with social media links, logout, and language links is aligned to the right.
Let's compare and fix them: 👇 
- Your design: 👇 
![homepage-1](https://user-images.githubusercontent.com/67087664/146963159-87667af7-40cd-45d5-9359-a832c0521701.png)
- Original: 👇 
![homepage+1_LI](https://user-images.githubusercontent.com/67087664/146963249-ec2d2ce9-9c1b-451f-8ae2-02f346bf5278.jpg)

Reply:
It's done

<img width="1142" alt="Capture d’écran 2021-12-21 à 19 33 51" src="https://user-images.githubusercontent.com/7243132/146980982-65599a00-91d1-43c0-8519-c20316836759.png">